windows/src/wmi.cc

```diff
@@ -25,9 +27,9 @@ namespace leatherman { namespace windows {
     {
 #ifdef LEATHERMAN_I18N
         // LOCALE: format a pointer as hex for printing an error message.
-        return lth_locale::format("{1} (0x{2,num=hex})", s, hres);
+        return _("{1} (0x{2,num=hex})", s, hres);
 #else
-        return lth_locale::format("%1% (0x%2$#x)", s, hres);
+        return _("%1% (0x%2$#x)", s, hres);
```

Comment:
I think lth_locale does the right thing here now, and we shouldn't need to special-case format strings anymore